### PR TITLE
implemented constructor-pages container

### PR DESCRIPTION
**Pull request by Pipuss**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** original screenshot ** | ** updated screenshot ** |

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally and linter has no warnings and errors
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/containers/constructor-page-container/constructor-page-container.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    const deleteItem = () => {
+      dispatch(closeDialog());
+      dispatch(deleteItemAction(id));
+    };
+    openSuccessSnackbar(deleteItem, ITEM_REMOVE_MESSAGE);
+  };
+
+  const constructorItems = items.map((item) => {
+    const id = item._id;
+    const deleteHandler = () => itemDeleteHandler(id);
```
Comment (tarasSanotskyi) [inline src/containers/constructor-page-container/constructor-page-container.js:67]: it is probably better to move these functions from the callback to the top

File: src/containers/constructor-page-container/constructor-page-container.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+    const props = getTableRowProps(item, deleteHandler, editHandler);
+
+    return <TableContainerRow key={id} {...props} />;
+  });
+
+  return (
+    <>
+      {loading ? (
+        <LoadingBar />
```
Comment (tarasSanotskyi) [inline src/containers/constructor-page-container/constructor-page-container.js:78]: remove the ternary operator from the template

File: src/containers/constructor-page-container/constructor-page-container.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+              color={materialUiConstants.primary}
+            >
+              {CREATE_ITEM_TITLE}
+            </Button>
+          </div>
+          <div>
+            <FilterNavbar options={filterOptions} />
+          </div>
+          {constructorItems.length ? (
+            <TableContainerGenerator
```
Comment (tarasSanotskyi) [inline src/containers/constructor-page-container/constructor-page-container.js:99]: remove the ternary operator from the template2

File: src/containers/constructor-page-container/constructor-page-container.js (hunk 1 above)
Comment (Pipuss) [reply src/containers/constructor-page-container/constructor-page-container.js:67]: Probably, but not in this case

Comment (tarasSanotskyi) [reply src/containers/constructor-page-container/constructor-page-container.js:67]: ok

**Review by tarasSanotskyi — APPROVED** [review]

**Review by tarasSanotskyi — DISMISSED** [review]

**Review by Tolik170 — DISMISSED** [review]

**Review by wonderflu — APPROVED** [review]

**Review by rostyslaw27 — APPROVED** [review]

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/configs/button-titles.js [hunk 4]
```diff
@@ -81,16 +81,32 @@ const buttonTitles = {
   MODEL: 'Моделі',
   YES_BUTTON: 'Так',
   NO_BUTTON: 'Ні',
-  CREATE_BACK_TITLE: 'Додати спинку',
-  CREATE_BOTTOM_TITLE: 'Додати низ',
-  CREATE_POCKETS_TITLE: 'Додати кишеню',
+  constructorPageTitles: {
```
[the remark this answers is not in the document]
Comment (Pipuss) [reply src/configs/button-titles.js]: Fixed

**Review by Tolik170 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**